app/views/event_attendances/index.html.erb

```diff
@@ -1,91 +1,123 @@
+
+        <%- if @event.attendees.empty? -%>
+            <tr class="empty">
+                <td colspan=10>No event attendees yet.</td>
+            </tr>
+        <% end -%>
+    </tbody>
 </table>
 
 <div id="sidebar">
-	<p><%= link_to "Back to Event", event_path(@event), :class => 'back button' %></p>
-	<p><%= link_to "Take Attendnace", checkin_event_event_attendances_path(@event, :audience => @audience), :class => "edit button" %></p>
-	
-	<%- form_tag({}, :method => :get) do -%>
-		<p><%= label_tag :audience, "View Attendance For:" %>
-			<%= select_tag :audience,
-							options_for_select(%w[Participant Student Mentor Volunteer].collect{|a| ["#{a} (#{@event.rsvpd_count(a)})", a]}, 
-							@audience.to_s.classify),
-							:onchange => "this.form.submit()" %>
-			<%= submit_tag "Go" %>
-		</p>
-	<% end -%>
-
-	<%- if @event.has_shifts?(@audience.to_s.classify) %>
-		<h3>Filter by Shift</h3>
-		
-		<div id="event_shift_selectors">
-		<%- for event_shift in @event.shifts.for(@audience.to_s.classify) -%>
-			<label for="event_shift_selector_<%= event_shift.id %>">
-				<input type="checkbox" id="event_shift_selector_<%= event_shift.id %>" data-shift-id="<%= event_shift.id %>" checked onChange="updateShiftSelections(this);">
-				<%= truncate(event_shift.title) %>
-			</label><br>
-		<% end -%>
-		<small class="light">Filtering the list will also update the email buttons above.</small>
-		</div>
-		
-	<% end -%>
+    <p><%= link_to "Back to Event", event_path(@event), :class => 'back button' %></p>
+    <p><%= link_to "Take Attendnace", checkin_event_event_attendances_path(@event, :audience => @audience), :class => "edit button" %></p>
+    
+    <%- form_tag({}, :method => :get) do -%>
+        <p><%= label_tag :audience, "View Attendance For:" %>
+            <%= select_tag :audience,
+                            options_for_select(%w[Participant Student Mentor Volunteer].collect{|a| ["#{a} (#{@event.rsvpd_count(a)})", a]}, 
+                            @audience.to_s.classify),
+                            :onchange => "this.form.submit()" %>
+            <%= submit_tag "Go" %>
+        </p>
+    <% end -%>
+
+    <%- if @event.has_shifts?(@audience.to_s.classify) %>
+        <h3>Filter by Shift</h3>
+        
+        <div id="event_shift_selectors">
+        <%- for event_shift in @event.shifts.for(@audience.to_s.classify) -%>
+            <label for="event_shift_selector_<%= event_shift.id %>">
+                <input type="checkbox" id="event_shift_selector_<%= event_shift.id %>" data-shift-id="<%= event_shift.id %>" checked onChange="updateShiftSelections(this);">
+                <%= truncate(event_shift.title) %>
+            </label><br>
+        <% end -%>
+        <small class="light">Filtering the list will also update the email buttons above.</small>
+        </div>
+        
+    <% end -%>
 </div>
 
 <script type="text/javascript">
 
+function filterEvents() {
```

Comment:
I think this can be deleted. I didnt even see this empty stub... I will delete this asap